jena-shex/src/main/java/org/apache/jena/shex/ShapeMap.java

```diff
@@ -29,32 +29,32 @@
  * <a href="https://shexspec.github.io/shape-map/">ShEx shape map</a> used for both
  * targeting validation and reporting violations.
  */
-public class ShexMap {
+public class ShapeMap {
 
```

Comment:
Small problem.

This is API. To migrate, and be compatible with return types, so make ShapeMap a subclass of ShexMap; deprecate ShexMap. After the next release, thus can be cleaned up and ShexMap removed.

There may need to be multiple methods where the argument is a ShexMap. Can't do both with sub/super classes.